README.md

```diff
@@ -26,6 +26,8 @@ For more information, refer to the [mining guide](https://docs.arweave.org/info/
 - CMake
 - SQLite3 headers (libsqlite3-dev on Ubuntu)
 
+Install all of these in one line on Ubuntu with `sudo apt install erlang build-essential clang cmake libsqlite3-dev`
```

Comment:
How about something like?

You can install all of these dependencies on Ubuntu using the following command:
```
sudo apt install erlang build-essential clang cmake libsqlite3-dev
```